lib/PlacesProvider.js

```diff
@@ -475,9 +475,9 @@ Links.prototype = {
 /**
  * Singleton that serves as the default link provider for the grid.
  */
-const gLinks = new Links();
+const gLinks = new Links(); // eslint-disable-line no-use-before-define
```

Comment:
I don't know how to solve this. Currently this little line is causing about 10 warnings from ESLint:
```
/Users/pdehaan/dev/github/activity-streams/lib/PlacesProvider.js
   87:7   warning  'gLinks' was used before it was defined  no-use-before-define
   93:7   warning  'gLinks' was used before it was defined  no-use-before-define
  101:9   warning  'gLinks' was used before it was defined  no-use-before-define
  113:7   warning  'gLinks' was used before it was defined  no-use-before-define
  117:7   warning  'gLinks' was used before it was defined  no-use-before-define
  134:9   warning  'gLinks' was used before it was defined  no-use-before-define
  135:11  warning  'gLinks' was used before it was defined  no-use-before-define
  142:9   warning  'gLinks' was used before it was defined  no-use-before-define
  148:9   warning  'gLinks' was used before it was defined  no-use-before-define
  149:11  warning  'gLinks' was used before it was defined  no-use-before-define
```

I couldn't find the optimal place to put this magic little line where `Links` was defined and tests would still pass. And I didn't want to solve it by adding 10 `// eslint-disable-line no-use-before-define` hacks all over the place.